app/stacks/uk-west/applications-service/terragrunt.hcl

```
@@ -60,6 +61,7 @@ inputs = {
   back_office_service_bus_nsip_project_update_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_project_update_topic_id
   back_office_service_bus_nsip_representation_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_representation_id
   back_office_service_bus_nsip_exam_timetable_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_exam_timetable_id
+  back_office_service_bus_nsip_service_user_topic_id            = dependency.back_office_ukw.outputs.service_bus_nsip_service_user_topic_id

```

```suggestion
  back_office_service_bus_nsip_service_user_topic_id            = dependency.back_office_ukw.outputs.servicebus_topic_service_user_id
```

I think the output should be named as above. The back-office outputs are here: `app/stacks/uk-west/back-office/outputs.tf`